specifyweb/frontend/js_src/lib/components/Molecules/DateElement.tsx

```diff
@@ -19,14 +19,42 @@ export function DateElement({
   readonly fallback?: React.ReactNode;
   readonly flipDates?: boolean;
 }): JSX.Element {
+  const [relativeDate, setRelativeDate] = React.useState<string | undefined>(
+    date ? getRelativeDate(new Date(date)) : undefined
+  );
+
+  React.useEffect(() => {
+    let intervalValue;
+    let interval: NodeJS.Timeout | undefined;
+
+    if (date) {
+      interval = setInterval(() => {
+        const dateObject = new Date(date);
+        const now = new Date();
+        const timeDifference = now.getTime() - dateObject.getTime();
+
+        if (timeDifference < 60000) intervalValue = 5000;
```

Comment:
can you please refactor to use these time units instead of numbers?
https://github.com/specify/specify7/blob/xml-editor/specifyweb/frontend/js_src/lib/components/Atoms/timeUnits.ts

<details><summary>Hint</summary>
<p>

i.e, `60 * MILLISECONDS`
or `HOUR * MILLISECONDS`

</p>
</details> 

much more readable and less bug prone this way